.github/workflows/run_fvt_tests.yaml

```diff
@@ -91,6 +94,10 @@ jobs:
       run: |
         set -x
         IMAGE=emqx/${{ matrix.profile }}:$(./pkg-vsn.sh)
+        if [[ "${{ matrix.profile }}" = *-elixir ]]
```

Comment:
Are we building/checking elixir version only for CE (emqx-elixir)?

Reply:
Currently, yes.  I've added only the CE edition, since we want only a few things with Elixir at this moment (only zips, no packages, one docker image).